Makefile

```diff
@@ -63,10 +65,3 @@ docker:
 		--build-arg GITVERSION=$(GITVERSION) \
 		-t $(IMAGE_NAME) .
 	@echo built image $(IMAGE_NAME)
-
-COMPONENT_TEST_IMAGE_NAME ?= docker.io/vshn/$(BINARY_NAME)-component-test:$(VERSION)
-
-.PHONY: docker-component-test
-docker-component-test:
-	docker build -t $(COMPONENT_TEST_IMAGE_NAME) -f tools/Dockerfile.component_test .
-	@echo built image $(COMPONENT_TEST_IMAGE_NAME)
```

Comment:
This change doesn't need to be in this PR anymore, this already has been removed in #127

Reply:
Sadly we missed that one in #127 so it has to stay.